tests/integration/targets/test_backup_create_plan/tasks/main.yml

```
@@ -0,0 +1,125 @@
+---
+- name: Set 'test_resource_prefix' variable
+  ansible.builtin.set_fact:
+    test_resource_prefix: backup-resource-test-{{ 101 | random(1, 10) }}
```

you can use `cloud/aws` integration tests resource prefix
```suggestion
    test_resource_prefix: backup-resource-test-{{ resource_prefix }}
```